inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -376,7 +376,23 @@ std::vector<std::vector<std::uint8_t>> LayerTestsCommon::CalculateRefs() {
         refInputsTypes[i] = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(memory->getTensorDesc().getPrecision());
     }
 
-    const auto &&outputsInfo = executableNetwork.GetOutputsInfo();
+    auto mapInfo = executableNetwork.GetOutputsInfo();
+    std::vector<std::pair<std::string, InferenceEngine::CDataPtr>> outputsInfo(mapInfo.begin(), mapInfo.end());
+    typedef std::pair<std::string, InferenceEngine::CDataPtr> pp;
+    struct {
+      bool operator()(
+          const pp a,
+          const pp b) const {
+        bool result = false;
+        if (a.first.length() < b.first.length()) {
+          result = true;
+        } else if (a.first.compare(b.first) < 0) {
+          result = true;
+        }
+        return result;
+      }
+    } blobLess;
```

Comment:
I'm a bit skeptical about the name `blobLess`, I'm not sure how "blob" is related to this comparison (perhaps I'm missing something?). I suggest `alphabeticalLess` or `lexicalLess` or `nameLess` instead.

Reply:
Done, use nameLess, we want to compare output name here ( is blob),  for example , let the "xxx-8" to be placed before "xxx-10", but it has opposite order with traditional string comparison,  so nameLess may be better here.

Reply:
Ah, thanks for the details, I didn't catch the reversed order. In that case, this name is fine, but reverseNameLess might be even better.